File: Data Structures & Algorithms/duplicate-integer/submission-4.java
class Solution 
{
    public boolean hasDuplicate(int[] nums) 
    {
        for(int x = 0; x < nums.length; x++)
        {
            for (int y = x + 1 ; y < nums.length; y++)
            {
                if (nums[x] == nums[y])
                {
                    return true;
                }
            }
        }
        return false; 
    }

}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-4.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        
        count = {}
        res = []

        for n in nums:
            count[n] = 1 + count.get(n,0)
        
        freq = [[] for i in range(len(nums) + 1)]

        for c,num in count.items():
            freq[num].append(c)
        
        for i in range(len(freq) - 1, -1, -1):
            for num in freq[i]:
                res.append(num)
                if len(res) == k:
                    return res

